manager/main.go

```diff
@@ -55,7 +56,7 @@ func init() {
 
 func run(namespace string, metricsAddr string, enableLeaderElection bool,
 	enableApplicationController, enableBlueprintController, enablePlotterController bool) int {
-	setupLog.Trace().Msg("creating manager")
+	setupLog.Trace().Msg("creating manager. based on git commit: " + gitCommit)
```

Comment:
sorry for the late comment, just put attention. 
please change the log level from Trace to Info. The used commit information can be useful for investigations of production failures.